src/components/Footer.js

```diff
@@ -0,0 +1,56 @@
+import Link from 'next/link';
+import {useRouter} from 'next/router';
+import styled from 'styled-components';
+
+const StyledFooter = styled.footer`
+	position: fixed;
+	bottom: 0;
+	left: 0;
+	width: 100%;
+	height: 60px;
+	background: #f01878;
+`;
+
+const StyledNav = styled.nav`
+	display: flex;
+	padding: 15px;
+	background-color: #f01878;
+	color: white;
+`;
+
+const StyledLink = styled.a`
+	padding: 0 200px;
```

Comment:
Such a large padding looks really unusual to me 🤔 If you have a look at your app, the navigation is not really responsive. As your `nav` element is a flexbox, you can align the items with the following:

```css
justify-content: space-evenly;
```

Reply:
Ok, erledigt.